frontend/mobilityai_react/App/Lib/Api.js

```diff
@@ -76,8 +76,8 @@ export function GetPatientAchievements(patientId) {
         .catch((error) => console.log(JSON.stringify(error)));
 }
 
-export function AddPatientAchievements(patientId, steps, activeMinutes, walkingMinutes, standingMinutes) {
-    const route = "api/Patients/" + patientId + "/Achievements?steps=" + steps + "&activeMinutes=" + activeMinutes + "&walkingMinutes=" + walkingMinutes + "&standingMinutes=" + standingMinutes;
+export function AddPatientAchievements({id, steps, activeTime, walking, standing, ...props}) {
```

Comment:
Previously, all 5 were being passed in as parameters, however, these are all part of the state, you can do object destructuring to make it so the function has one param instead of 5. I pass in the props because I pass in the state object and it has extra keys besides the ones I am taking.